packages/product-feed-zbozi/tests/Unit/ZboziFeedItemTest.php

```diff
@@ -0,0 +1,229 @@
+        $availabilityMock->method('getDispatchTime')->willReturn(0);
+        $this->defaultProduct->method('getCalculatedAvailability')->willReturn($availabilityMock);
+
+        $productPrice = new ProductPrice(new Price(0, 0), false);
+        $this->productPriceCalculationForUserMock->method('calculatePriceForUserAndDomainId')
+            ->with($this->defaultProduct, 1, null)->willReturn($productPrice);
+
+        $this->productUrlsBatchLoaderMock->method('getProductUrl')
+            ->with($this->defaultProduct, $this->defaultDomain)->willReturn('https://example.com/product-1');
+
+        $this->categoryFacadeMock->method('getCategoryNamesInPathFromRootToProductMainCategoryOnDomain')
+            ->with($this->defaultProduct, $this->defaultDomain)->willReturn(['category A', 'category B', 'category C']);
+    }
+
+    /**
+     * @param int $id
+     * @param string $url
+     * @param string $locale
+     * @return \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig|\PHPUnit\Framework\MockObject\MockObject
+     */
+    private function createDomainConfigMock(int $id, string $url, string $locale): DomainConfig
+    {
+        $domainConfigMock = $this->createMock(DomainConfig::class);
+
+        $domainConfigMock->method('getId')->willReturn($id);
+        $domainConfigMock->method('getUrl')->willReturn($url);
+        $domainConfigMock->method('getLocale')->willReturn($locale);
+
+        return $domainConfigMock;
+    }
+
+    public function testMinimalZboziFeedItemIsCreatable()
+    {
+        $zboziFeedItem = $this->zboziFeedItemFactory->create($this->defaultProduct, null, $this->defaultDomain);
+
+        self::assertInstanceOf(ZboziFeedItem::class, $zboziFeedItem);
+
+        self::assertEquals(1, $zboziFeedItem->getId());
+        self::assertEquals(1, $zboziFeedItem->getSeekId());
+        self::assertNull($zboziFeedItem->getGroupId());
+        self::assertEquals('product name', $zboziFeedItem->getName());
+        self::assertNull($zboziFeedItem->getDescription());
+        self::assertEquals('https://example.com/product-1', $zboziFeedItem->getUrl());
+        self::assertNull($zboziFeedItem->getImgUrl());
+        self::assertEquals(0, $zboziFeedItem->getPrice()->getPriceWithoutVat());
+        self::assertEquals(0, $zboziFeedItem->getPrice()->getPriceWithVat());
+        self::assertNull($zboziFeedItem->getEan());
+        self::assertNull($zboziFeedItem->getProductno());
+        self::assertEquals(0, $zboziFeedItem->getDeliveryDate());
+        self::assertNull($zboziFeedItem->getManufacturer());
+        self::assertEquals('category A | category B | category C', $zboziFeedItem->getCategoryText());
+        self::assertEquals([], $zboziFeedItem->getParams());
+        self::assertNull($zboziFeedItem->getMaxCpc());
+        self::assertNull($zboziFeedItem->getMaxCpcSearch());
+    }
+
+    public function testZboziFeedItemWithGroupId()
```

Comment:
same problem as I mentioned for `HeurekaFeedItemTest` - the content of the method does not correspond with it's name

Reply:
Fixed in c30a011b0